src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,20 @@
-export const UserInfo = () => {};
+import React from 'react';
+import { User } from '../../types/User';
+
+interface UserInfoProps {
+  user: User | null;
+}
+
+export const UserInfo: React.FC<UserInfoProps> = ({ user }) => {
+  if (!user) {
+    return <p>No user found</p>;
+  }
```

Comment:
```suggestion
  user: User;
}

export const UserInfo: React.FC<UserInfoProps> = ({ user }) => {
```
You don't need to render `UserInfo` component at all if there is no user. Do this check in the `TodoInfo` component